fedoracommunity/connectors/bodhiconnector.py

```diff
@@ -579,21 +585,25 @@ def query_active_releases(self, filters=None, **params):
                     if query.endswith('-testing'):
                         nvr = parse_build(release['nvr'])
                         row['testing_version'] = HTML.tag(
-                            'a', c='%(version)s-%(release)s' % nvr,
-                            href='%s/updates/%s' % (self._prod_url, nvr['nvr']))
+                            'a',
+                            c='%(version)s-%(release)s' % nvr,
+                            href=koji_build_url % nvr)
```

Comment:
We are.  But like I said, this links to the koji build directly for all build nvrs.
And then additionally, later, an extra link is tacked on to add a link to the bodhi update.